src/main/java/co/cask/plugin/format/DBTypes.java

```diff
@@ -124,8 +132,14 @@
     return type;
   }
 
+  private static String formatDate(long time, String format) {
+		DateFormat dateFormat = new SimpleDateFormat(format);
```

Comment:
the date format will never change, so it's better to create the DateFormat once and pass it into transformValue instead of having this method